.github/workflows/build-and-push-to-ghcr.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           push: false # Do not push image for PR
           cache-from: type=registry,ref=ghcr.io/coqui-ai/xtts-streaming-server:cache-pr-${{ github.event.pull_request.head.sha }}
           cache-to: type=registry,ref=ghcr.io/coqui-ai/xtts-streaming-server:cache-pr-${{ github.event.pull_request.head.sha }}
-          tags: coqui-ai/xtts-streaming-server:${{ env.RELEASE_VERSION }}-${{ github.sha }}
+          tags: coqui-ai/xtts-streaming-server:pr-${{ github.event.pull_request.head.sha }}
```

Comment:
Should remove the `tags:` input here (since we're not pushing) and use the PR number instead of HEAD commit hash in the cache lines above, as it changes on every push, breaking the caching.

Reply:
HEAD is the latest, not the first